src/Swarm/Language/Capability.hs

```diff
@@ -136,27 +59,8 @@ data Capability
     CLambda
   | -- | Enable recursive definitions
     CRecursion
-  | -- | Execute the 'Reprogram' command
-    CReprogram
-  | -- | Execute the `meet` and `meetAll` commands.
-    CMeet
   | -- | Capability to introspect and see its own name
     CWhoami
-  | -- | Capability to set its own name
-    CSetname
-  | -- | Capability to move unrestricted to any place
-    CTeleport
-  | -- | Capability to run commands atomically
-    CAtomic
-  | -- | Capability to execute swap (grab and place atomically at the same time).
-    CSwap
-  | -- | Capability to obtain absolute time, namely via the `time` command.
-    CTimeabs
-  | -- | Capability to utilize relative passage of time, namely via the `wait` command.
-    --   This is strictly weaker than "CTimeAbs".
```

Comment:
I would like to get rid of these confusing capabilities that say they do more, when it is the entities that do the work.

The `CTimerel` capability _is not strictly weaker itself_, this logic is in entities providing it along with `CTimeabs`.

The idea would be to specify all the commands that entities want to enable in the capability list.

@byorgey @kostmo what do you think? 🤔

Reply:
We should just change `CTimeabs` into `CExecute Time` and `CTimerel` into `CExecute Wait`, I think.  See my other comment below.